CryptoAnalysis/src/main/java/crypto/reporting/ErrorMarkerListener.java

```diff
@@ -54,24 +55,44 @@
 	protected final Table<SootClass, SootMethod, Set<AbstractError>> errorMarkers = HashBasedTable.create(); 
 	protected final Map<Class, Integer> errorMarkerCount = new HashMap<Class, Integer>();
 	protected final List<IAnalysisSeed> secureObjects = new ArrayList<IAnalysisSeed>();
+	protected List<String> ignorePackagesList =  Collections.<String>emptyList();
 	
 	private void addMarker(AbstractError error) {
 		SootMethod method = error.getErrorLocation().getMethod();
 		SootClass sootClass = method.getDeclaringClass();
-		
 		Set<AbstractError> set = errorMarkers.get(sootClass, method);
+		Integer integer = errorMarkerCount.get(error.getClass());
```

Comment:
Rename variable to more descriptive name. I have a hard time understanding what this variable is doing.

Reply:
agree

Reply:
Renamed integer to errorCount.